Library/Homebrew/tap.rb

```diff
@@ -201,7 +201,9 @@ def core_tap?
   # install this {Tap}.
   #
   # @param [Hash] options
-  # @option options [String]  :clone_targe If passed, it will be used as the clone remote.
```

Comment:
While I was in here, I went ahead and fixed the spelling on this option to match the code.